recipes/libradolan/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+src_dir="$(pwd)"
+mkdir ../build
+cd ../build
+cmake $src_dir -DCMAKE_INSTALL_PREFIX=$PREFIX
+make
+make install
```

Comment:
Is there a `make check` or similar?

Can you also add a comment about the build? (It is unusual to have a two part build with `cmake` and then `make`.)